evostar2019/angrybirds-evo19.tex

```diff
@@ -1185,70 +1144,71 @@ \section{Conclusions and Future Work}
 results are variations of the same idea. However, it is crystal clear
 from the experiments run in this project that a lack of knowledge
-will lead to unexpected outcomes. In our case, the fitness function used the 
-stability of the structure and only considered other features--- overlapping 
-blocks and distance to the ground--- to ensure the levels would be valid.
- % I am lost in this paragraph. It should be more
- % related to the actual experiments made - JJ
-
-The second objective, adapting the game to extract data from execution, was 
-certainly achieved. It was also a basic requirement to proceed with the rest of 
-them. The game does provide the data, as long as the input is correctly 
-structured. Although the original intention was to make it available on Linux, 
-currently it can only be executed with the desired behaviour on Windows. 
-Another issue is that the simulation is not easily adaptable. If the fitness 
-function of the EA is changed and needs data that is not included in the output 
-right now, the game would have to be changed and compiled again. It
-would be interesting to obtain other kind of data from the simulator,
-such as the height of the structure and, eventually, its resistance to
-bombardment by angry birds. However, this is left as future work.
-
-Producing stable structures under gravity was the third objective and
-the closest to the ultimate one. That objective was effectively
-achieved, but the consequence of evolving in a path of minimum
+will lead to unexpected and even disappointing outcomes. % I am lost
+                                % in this paragraph. It should be more
+                                % related to the actual experiments
+                                % made - JJ
+% Antonio - I have reduced and focused a bit more the text
+
+In order to conduct our experiments the Angry Birds simulator (Science Birds) has been adapted to our necessities, yielding now some other information required to evaluate the individuals of the implemented EA.
+
+%The second objective, adapting the game to extract data from execution, was 
+%certainly achieved. It was also a basic requirement to proceed with the rest of 
+%them. The game does provide the data, as long as the input is correctly 
+%structured. Although the original intention was to make it available on Linux, 
+%currently it can only be executed with the desired behaviour on Windows. 
+%Another issue is that the simulation is not easily adaptable. If the fitness 
+%function of the EA is changed and needs data that is not included in the output 
+%right now, the game would have to be changed and compiled again. It
+%would be interesting to obtain other kind of data from the simulator,
+%such as the height of the structure and, eventually, its resistance to
+%bombardment by angry birds. However, this is left as future work.
+
+% Antonio - Again, in a paper conclusions you must remark what you have done. ;)
```

Comment:
It's conclusion and discussion. We can say what we have achieved, and why, and what we have not, and why.